source/change-streams/change-streams.rst

```diff
@@ -836,7 +836,7 @@ All drivers MUST document how users can iterate a change stream and receive *all
 Timeouts
 ^^^^^^^^
 
-Drivers MUST apply timeouts to change stream establishment, iteration, and resume attempts per `Client Side Operations Timeout: Change Streams <../client-side-operations-timeout/client-side-operations-timeout.rst#Change-Streams>`__.
+Drivers MUST apply timeouts to change stream establishment, iteration, and resume attempts per `Client Side Operations Timeout: Change Streams <../client-side-operations-timeout/client-side-operations-timeout.md#Change-Streams>`__.
```

Comment:
```suggestion
Drivers MUST apply timeouts to change stream establishment, iteration, and resume attempts per `Client Side Operations Timeout: Change Streams <../client-side-operations-timeout/client-side-operations-timeout.md#change-streams>`__.
```

I think all of the generated headings are lowercase, unless GitHub creates multiple variations for each heading. I'm checking this by viewing the rendered copy of the ported CSOT spec (e.g. [client-side-operations-timeout.md](https://github.com/mongodb/specifications/blob/1d074a35ab9bcd4496aae82ced5ae0b3a0ebd62f/source/client-side-operations-timeout/client-side-operations-timeout.md)).